projects/fsastorefrontstyles/scss/base/building-blocks.scss

```diff
@@ -237,3 +237,20 @@
     border-right: 7px solid transparent;
   }
 }
+
+.currency {
+  .currencyCode {
+    border: 1px solid #d3d6db;
+    border-right: 0;
+    border-top-left-radius: 0.25rem;
+    border-bottom-left-radius: 0.25rem;
+    height: 48px;
```

Comment:
not needed since BS input has margin-bottom of 0.25rem, you should do the same and the elements should align.